packages/design-system/.storybook/docs/tokensDocHelpers/components/DefinitionList/DefinitionListItem/DefinitionListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+import { Token } from '../../../../../../src/tokens/types';
+import styles from './DefinitionListItem.module.scss';
+import TokenName from '../../../TokenName';
+import { getCssName, getScssName } from '../../../TokenFormatter';
+import CopyValue from './CopyValue';
+import classnames from 'classnames';
+
+function DefinitionListItem({ token, children }: { token: Token; children?: ReactElement }) {
+	if (!token) {
+		return null;
+	}
+
+	return (
+		<div className={classnames(styles.listItem, { [styles.listItem__noCard]: !children })}>
+			{children && <dd className={styles.listItem__demo}>{children}</dd>}
+
+			<div className={styles.listItem__description}>
+				<dt className={styles.tokenName}>
```

Comment:
Not sure that it's valid html. 
`dl > div > dt` is valid
`dl > div > div> dt` is not

[Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt)

Reply:
Ah! Mmh. My bad then. I'll remove de dl's. They bring very little but made Fabien happy

Reply:
https://www.stefanjudis.com/today-i-learned/divs-are-valid-elements-inside-of-a-definition-list/ it's hard to tell 🤔

Reply:
1 level of div yes, but not multiple.
I saw on MDN that the parent of dt can be a div under a dl. And I tried an online spec checker, it returned an error with multiple level of divs

Reply:
And using dl for definitions is a good practice for a11y, AND it makes Fabien happy :)